README.md

```diff
@@ -70,6 +70,26 @@ module "observability_agents" {
 }
 ```
 
+### Customise Kubernetes metadata line filtering in the LogDNA agent
+
+This module allows to customise the Kubernetes metadata line filtering in the LogDNA agent by setting the value of the input parameters `logdna_agent_custom_line_exclusion` and `logdna_agent_custom_line_inclusion`. For more information, see [Configuration for Kubernetes Metadata Filtering](https://github.com/logdna/logdna-agent-v2/blob/3.8/docs/KUBERNETES.md#configuration-for-kubernetes-metadata-filtering)
+
+For example, you can set the following input values for the parameters:
+
+```text
+custom_logdna_at_agent_line_exclusion = "label.app.kubernetes.io/name:sample-app\\, annotation.user:sample-user"
+custom_logdna_at_agent_line_inclusion = "namespace:default"
+```
+
+The resulting daemonset configuration will be:
+
+```text
+- name: LOGDNA_K8S_METADATA_LINE_INCLUSION
+    value: "label.app.kubernetes.io/name:sample-app, annotation.user:sample-user"
+- name: LOGDNA_K8S_METADATA_LINE_EXCLUSION
```

Comment:
Suggestion:

## Configuration for Kubernetes metadata filtering in the LogDNA agent

You can configure the LogDNA agent to filter log lines that are associated with Kubernetes resources by setting the exclusion and inclusion parameters.

For example, to set the agent to return all log lines coming from the "default" Kubernetes and exclude anything with a value of `app.kubernetes.io/name:sample-app` or an annotation with `user:sample-user`, include these parameters:

```text
custom_logdna_at_agent_line_exclusion = "label.app.kubernetes.io/name:sample-app\\, annotation.user:sample-user"
custom_logdna_at_agent_line_inclusion = "namespace:default"
```

The following is the corresponding DaemonSet configuration:

```text
- name: LOGDNA_K8S_METADATA_LINE_INCLUSION
  value: "label.app.kubernetes.io/name:sample-app, annotation.user:sample-user"
- name: LOGDNA_K8S_METADATA_LINE_EXCLUSION
  value: "namespace:default"
```

For more information, see [Configuration for Kubernetes Metadata Filtering](https://github.com/logdna/logdna-agent-v2/blob/3.8/docs/KUBERNETES.md#configuration-for-kubernetes-metadata-filtering).

Reply:
Set heading to H2 (`##`).

Move the more info after your description of what to do.

I rewrote a bit based on the link you provided. See if it is still correct.

Reply:
I tried to adopt your suggestion by mentioning the k8s metadata filtering